main.tf

```diff
@@ -1,11 +1,13 @@
 locals {
-  account_id                        = var.existing_watsonx_orchestrate_instance_crn != null ? module.crn_parser[0].account_id : ibm_resource_instance.watsonx_orchestrate_instance[0].account_id
-  watsonx_orchestrate_id            = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].id : ibm_resource_instance.watsonx_orchestrate_instance[0].id
-  watsonx_orchestrate_crn           = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].crn : ibm_resource_instance.watsonx_orchestrate_instance[0].crn
-  watsonx_orchestrate_guid          = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].guid : ibm_resource_instance.watsonx_orchestrate_instance[0].guid
-  watsonx_orchestrate_name          = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].resource_name : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_name
-  watsonx_orchestrate_plan_id       = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_plan_id
-  watsonx_orchestrate_dashboard_url = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].dashboard_url
+  account_id                  = var.existing_watsonx_orchestrate_instance_crn != null ? module.crn_parser[0].account_id : ibm_resource_instance.watsonx_orchestrate_instance[0].account_id
+  watsonx_orchestrate_id      = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].id : ibm_resource_instance.watsonx_orchestrate_instance[0].id
+  watsonx_orchestrate_crn     = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].crn : ibm_resource_instance.watsonx_orchestrate_instance[0].crn
+  watsonx_orchestrate_guid    = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].guid : ibm_resource_instance.watsonx_orchestrate_instance[0].guid
+  watsonx_orchestrate_name    = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].resource_name : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_name
+  watsonx_orchestrate_plan_id = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_plan_id
+  raw_crn                     = "crn:v1:bluemix:public:watsonx-orchestrate:${var.region}:a/${local.account_id}:${local.watsonx_orchestrate_guid}::"
```

Comment:
As we are not reading dashboard_url from data source, we do not need `raw_crn` and can use `local.watsonx_orchestrate_crn`